### ISSUE TITLE
[CRASH] Crash a few seconds after launch

### ISSUE THREAD

**Issue by Kore-Development** [therealfarfetchd/loadingspice#3]
[https://paste.ee/p/szgeC](https://paste.ee/p/szgeC)

**Comment by Kore-Development** [comment]
I just updated to the latest Fabric.

**Comment by Kore-Development** [comment]
I also put this in the Curseforge comments because I didn't know where to put the crash.

**Comment by dblsaiko** [comment]
Yeah, I know. It's broken because of multithreaded loading, which basically requires me to rewrite huge parts of it. And Minecraft has a loading bar now anyways, so this is not really needed anymore.
